src/controllers/stallController.ts

```diff
@@ -90,6 +91,9 @@ async function getCategories(stall: Stall) {
  * @param stall Stall instance
  */
 async function fmtStallResp(stall: Stall) {
+  const hawkerCentre = stall.HawkerCentre;
```

Comment:
Should do `stall!.HawkerCentre` here so that others are aware that any `stall` that comes in should already have their `hawkerCentre` retrieved.

Reply:
Added, thanks